pkgs/games/gnushogi/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchurl, zlib }:
+
+stdenv.mkDerivation rec {
+  name = "gnushogi-${version}";
+  version = "1.4.2";
+  buildInputs = [ zlib ];
+
+  src = fetchurl {
+    url = "https://ftp.gnu.org/gnu/gnushogi/${name}.tar.gz";
```

Comment:
If possible, we prefer to use mirror://gnu/ here.

Reply:
Oh, right, changed! :)